README.md

```diff
@@ -16,3 +16,157 @@ Template for PRM Deduction components, that can be used to make new components,
+| build         | Builds the app using babel into a `build/` folder                                                                        |
+| start         | Builds the app before launching the Express server on port `3000`                                                        |
+| start:local   | Builds the app locally (on `127.0.0.1:3000`)                                                                             |
+| start:nodemon | Builds the app before launching the Express server on port `3000` using [nodemon](https://www.npmjs.com/package/nodemon) |
+| test:\*       | For test scripts see [test section](#tests).                                                                             |
+
+## Set up
+
+1. Run `npm install` to install all node dependencies.
+2. Create a .env file at the root of the directory
+3. Copy the contents of the [.env.sample](./.env.sample) file at the root of the directory, and paste into the .env file. The .env.sample file contains template environment variables.
+
+- `NHS_ENVIRONMENT` - is set to the current environment ('dev' for OpenTest and 'test' for PTL environment) in which the container is deployed. It is populated by the pipeline.gocd.yml
+- `SERVICE_URL` -
+- `REPOSITORY_URI` - This is prepopulated by `tasks` (based on `IMAGE_REPO_NAME`)
+- `NODE_ENV` - set by the Docker files to be `local`
+- `AUTHORIZATION_KEYS` - a comma-separated list of Authorization keys. These are taken from AWS Parameters Store in the 'dev' and 'test' environments.
+
+4. If you would like to run locally, leave these blank, otherwise fill in the required fields.
+
+## Starting the app
+
+### Locally
+
+Docker image can be build locally with
+
+```
+./tasks build_docker_local
+```
+
+### Production mode
+
+Compile the code with `npm run build`, and then start the server with `npm start`.
+
+```bash
+# Dojo - same as what is run in pipeline
+
+# Builds the docker container with the app in
+./tasks build_docker_local
+
+# Runs the tests against the app in the docker container
+./tasks test_docker_local
+
+# Runs the ehr with db and localstack locally in interactive mode
+dojo -c Dojofile-dtest
+```
+
+## Swagger
+
+The swagger documentation for the app can be found at [http://localhost:3000/swagger](http://localhost:3000/swagger). To update it, change the
+`src/swagger.json` file. You can use [this editor](https://editor.swagger.io/) which will validate your changes.
+
+## Tests
+
+### Unit tests
+
+Run the unit tests with
+
+`npm test` or `./tasks test`
+
+### Integration tests
+
+Run the integration tests
+
+`npm run test:integration` or `./tasks test_integration`
+
+Bear in mind...
+
+### Coverage tests
+Runs the coverage tests (unit test and integration test) and collects coverage metrics
+
+`npm run test:coverage` or `./tasks test_coverage`
+
+### Docker tests
+
+Docker tests - to run the tests locally:
+
+```bash
+# Brings up the local test environment
+docker-compose up &
```

Comment:
I think these steps can be changed now to:
1. `npm run build`
2. `./tasks build_docker_local`
3. `./tasks test_docker_local`